README.md

```diff
@@ -116,7 +116,8 @@ VISUALIZATION_LAUNCHER = 'http://localhost:7000/vtk'
   UV-CDAT setup script.  If everything is setup correctly, you should be
   able to run the following:
   ```bash
-  ./vis_server/start.sh
+  cd vis_server
+  ./start.sh
```

Comment:
@doutriaux1 we should... once, I fix the main issue, I will fix some other minor things including this one.